CompetitionTemplate/src/competition/subsystems/drive/commands/UpdatePoseCommand.java

```diff
@@ -1,19 +1,23 @@
-package competition.subsystems.drive.commands;
+    package competition.subsystems.drive.commands;
 
 import com.google.inject.Inject;
 
 import competition.subsystems.drive.PoseSubsystem;
 import xbot.common.command.BaseCommand;
+import xbot.common.math.ContiguousDouble;
 
 
 public class UpdatePoseCommand extends BaseCommand {
 
     final PoseSubsystem pose;
+    final MonitorDefenseTraversalModule defenseTraversal;
     
     @Inject
-    public UpdatePoseCommand(PoseSubsystem pose)
+    public UpdatePoseCommand(PoseSubsystem pose, MonitorDefenseTraversalModule defenseTraversal)
     {
+        this.setRunWhenDisabled(true);
         this.pose = pose;
+        this.defenseTraversal = defenseTraversal;
```

Comment:
The name "defenseTraversal" doesn't really tell me what it is; when we are quickly trying to implement some new functionality, we want it to be clear.